x-pack/solutions/security/plugins/security_solution/public/common/components/filter_by_assignees_popover/filter_by_assignees_popover.test.tsx

```diff
@@ -110,14 +110,14 @@ describe('<FilterByAssigneesPopover />', () => {
     const onUsersChangeMock = jest.fn();
     const { getByTestId, getByText } = renderFilterByAssigneesPopover([], onUsersChangeMock);
 
-    getByTestId(FILTER_BY_ASSIGNEES_BUTTON).click();
+    fireEvent.click(getByTestId(FILTER_BY_ASSIGNEES_BUTTON));
```

Comment:
fixes [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/266993#0194655f-1467-452b-ae82-17015b38390e) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/266993/jobs/0194655f-1467-452b-ae82-17015b38390e/artifacts/01946583-11dc-44ac-8e59-602e4ce18840) Jest Tests #8 / should call onUsersChange on closing the popover

Looks like with an update the last event can be missed before the assertion. By using `fireEvent` we wrap each event in `act` allowing tests to process pending events.